bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/v2/RowMerger.java

```diff
@@ -60,12 +60,39 @@
  * rm.onComplete();
  * </pre>
  * <p>
- * When a complete row is found, {@link StreamObserver#onNext(Row)} will be called.
+ * When a complete row is found, {@link StreamObserver#onNext(Object)} will be called.
```

Comment:
RowMerger is an adapter.  It's a StreamObsever<ReadRowsResponse>, which calls a StreamObserver<Row>.  The intent of emitRow is to call StreamObserver<row>.onNext(Row).  I changed this to  {@link StreamObserver#onNext(Object)} since javadocs was giving a warning with the Generic parameter (Row).